appserver/web/web-glue/src/main/java/com/sun/enterprise/web/WebContainer.java

```diff
@@ -2058,7 +2061,7 @@ public void unloadWebModule(String contextRoot, String appName, String virtualSe
             }
         }
 
-        if (!hasBeenUndeployed) {
+        if (!hasBeenUndeployed && deployment.getCurrentDeploymentContext() == null) {
```

Comment:
Currently, this message is printed during deployment, and that's an error.
The message is intended to be printed during undeployment only.
This change will check if the deployment process is running, and if it is, it will inhibit this message.

The result is that the message will only be printed during "true" undeployment as it was intended to be originally.